packages/cli/src/main.ts

```diff
@@ -1,8 +1,68 @@
 #!/usr/bin/env node
-import { Command } from 'commander';
+import { Command, createOption, Option } from 'commander';
 import { node } from './commands/node';
+import { TaskParam, TaskArguments, initialKuai, ERRORS, KuaiError, paramTypes } from '@kuai/core';
+
+const KUAI_GLOBAL_PARAMS: Array<TaskParam> = [
+  {
+    name: 'config',
+    defaultValue: undefined,
+    description: 'A Kuai config file.',
+    type: paramTypes.path,
+    isFlag: false,
+    isOptional: true,
+  },
+];
+
+function parseTaskParams(param: TaskParam<TaskArguments>): Option {
+  return createOption(`--${param.name} <${param.type.name}>`, param.description);
+}
 
 const program = new Command();
 
+KUAI_GLOBAL_PARAMS.map((param) => parseTaskParams(param)).map((option) => program.addOption(option));
+
 program.addCommand(node);
+
+const config = () => {
+  let path;
+  for (let i = 0; i < process.argv.length; i++) {
+    if (process.argv[i] === '--config') {
+      if (i == process.argv.length - 1) {
+        throw new KuaiError(ERRORS.ARGUMENTS.MISSING_CONFIG_FILE);
+      }
+      path = process.argv[i + 1];
+      break;
+    }
+  }
```

Comment:
`process.argv.slice(0, -1).lastIndexOf('--config')` should work to check if the `config file` is specified.